src/containers/MyNdla/Arena/components/PostCard.tsx

```diff
@@ -277,25 +281,55 @@ const PostCard = ({ topic, post, onFollowChange, setFocusId, isMainPost, createR
     [created, language, timeDistance],
   );
 
+  const postUpvotes = useMemo(
+    () => (
+      <>
+        {post.owner?.id !== user?.id && (
+          <IconButtonV2
+            aria-label={post.upvoted ? t("myNdla.arena.posts.removeUpvote") : t("myNdla.arena.posts.upvote")}
+            title={post.upvoted ? t("myNdla.arena.posts.removeUpvote") : t("myNdla.arena.posts.upvote")}
+            variant="ghost"
+            colorTheme="light"
+            onClick={() =>
+              post.upvoted
+                ? removeUpvote({ variables: { postId: post.id } })
+                : upvote({ variables: { postId: post.id } })
+            }
+          >
+            {post.upvoted ? <ThumbFilled /> : <Thumb />}
+          </IconButtonV2>
+        )}
+        <TimestampText element="span" textStyle="content-alt" margin="none">
+          <span>{post.upvotes ?? 0}</span>
```

Comment:
Burde det være en alt-tekst her med "Antall som har likt innlegget" eller nokke sånt? No er det berre et tall.

Reply:
Det var egentlig planen, men sida du merga inn frontend packages PRen min med tekster, trudde jeg vi la den delen på is 😄 Kan legge inn tekster for det også i full fart, hvis du vil?

Reply:
Vi bør få dette publisert før vi tar inn jonas sine breaking ting ja